openui5/tsconfig.json

```diff
@@ -2,16 +2,10 @@
     "compilerOptions": {
         "module": "commonjs",
         "target": "es6",
-        "noImplicitAny": false,
+        "noImplicitAny": true,
         "strictNullChecks": false,
-        "types": [],
+        "types": ["jquery"],
```

Comment:
If I remove the reference it gives me an the error "error TS2688: Cannot find type definition file for 'jquery'.". How can I resolve this?